.github/workflows/unit-tests.yml

```diff
@@ -12,10 +12,11 @@ jobs:
       - run: |
           yarn install
           yarn prepack
+          export ENV=test-sfdx-plugin
```

Comment:
we test on studio-dev on purpose

Reply:
desicion changed to test on dedicated env